great_expectations/render/renderer_configuration.py

```diff
@@ -370,15 +370,26 @@ def _validate_for_row_condition(cls, values: dict) -> dict:
 
     @root_validator()
     def _validate_for_meta_notes(cls, values: dict) -> dict:
-        meta_notes: Optional[MetaNotes]
+        meta_notes: Optional[
+            dict[str, Optional[dict[str, list[str] | tuple[str] | str]]]
```

Comment:
`non-blocking`
It seems like there is some expected structure to this dict, can we use a TypedDict here?

Reply:
`MetaNotes` is a `TypedDict`, but we allow for more "loose" constraints at the API level and then this validator transforms the dict into a `MetaNotes` dict. I prefer to use the primitives here or else we end up with something like `MetaNotesInput` and `MetaNotes`.